Modules/Core/Transform/include/itkRigid3DPerspectiveTransform.hxx

```diff
@@ -152,10 +152,10 @@ Rigid3DPerspectiveTransform<TParametersValueType>::SetRotation(const Vector<TPar
   norm *= sinus;
   VnlQuaternionType q;
 
-  q[0] = cosinus;
-  q[1] = norm[0];
-  q[2] = norm[1];
-  q[3] = norm[2];
+  q[0] = norm[0];
+  q[1] = norm[1];
+  q[2] = norm[2];
+  q[3] = cosinus;
 
   VersorType v;
   v.Set(q);
```

Comment:
Perhaps it be clearer to directly use the [itk::Versor::Set](https://itk.org/Doxygen/html/classitk_1_1Versor.html#a352e896152c940d754f8f0c83ce3a83e) method which takes a vector, and an angle.